loops-and-arrays.js

```diff
@@ -65,15 +65,21 @@ const numbers = [2, 4, 5, 29, 38];
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
-    const squares = [30, 2, 8, 24, 11];
-    for (let i = 0; i < squares.length; i++) {
-        const volUme = (squares[i] ** 3);
-        console.log("Het volume van " + squares[i] + " is " + volUme);
-
+const squares = [30, 2, 8, 24, 11];
+for (let i = 0; i < squares.length; i++) {
+    squares[i] = "Het volume van " + squares[i] + " is " + squares[i] ** 3;
```

Comment:
Hi! Andere mogelijkheid had ook kunnen zijn:
for (let i = 0; i < squares.length; i++) {
  const volume = squares[i] * squares[i] * squares[i];

  squares[i] = "Het volume van " + squares[i] + " is " + volume;
}
Dit ziet er alsnog ook goed uit! Je hebt het veel makkelijker aangepakt door **3.